daemon/daemon_main.go

```diff
@@ -1576,6 +1570,13 @@ func initKubeProxyReplacementOptions() {
 
 	if !option.Config.EnableNodePort {
 		option.Config.EnableExternalIPs = false
+	} else {
+		if option.Config.Tunnel != option.TunnelDisabled &&
+			option.Config.NodePortMode != option.NodePortModeSNAT {
+
+			log.Warnf("Disabling NodePort's %q mode feature due to tunneling mode being enabled",
+				option.Config.NodePortMode)
```

Comment:
Missing `option.Config.NodePortMode = option.NodePortModeSNAT` to actually disable it?

Reply:
oh boy;-(